app/services/concerns/course/user_invitation_service/email_invitation_concern.rb

```diff
@@ -205,4 +206,16 @@ def validate_new_invitation_emails(invitations)
     end
     invitations
   end
+
+  # Validate that a row from the CSV file is valid.
+  # A row is considered valid if both fields are not blank and the email field
+  # contains an '@' symbol.
+  # TODO: Upgrade to a proper regex check for email.
+  #
+  # @param [Array] row An array of fields from a single row.
+  # @return [Boolean] True if the row is valid.
+  def valid_row?(row)
+    # The check for blanks MUST come first or the `include?` call might throw an exception.
+    !(row[0].blank? || row[1].blank?) && row[1].include?('@')
```

Comment:
Actually can use `row[0].present? && row[1].present? && row[1].include?('@')`

Can also include in the comments for this method to ensure that this is not the header row.